apps/native/app/src/screens/inbox/components/action-bar.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react'
+import styled, { useTheme } from 'styled-components/native'
+import { FormattedMessage } from 'react-intl'
+import { Image } from 'react-native'
+import Animated, { FadeInDown, FadeOutDown } from 'react-native-reanimated'
+
+import inboxReadIcon from '../../../assets/icons/inbox-read.png'
+import starIcon from '../../../assets/icons/star.png'
+import archiveIcon from '../../../assets/icons/tray.png'
+import { screenWidth } from '../../../utils/dimensions'
+import { Loader, Typography } from '../../../ui'
+
+const Wrapper = styled(Animated.View)`
+  height: 52px;
+  position: absolute;
+  bottom: ${({ theme }) => theme.spacing[2]}px;
+  right: ${({ theme }) => theme.spacing[2]}px;
+  border: 1px solid ${({ theme }) => theme.color.blue200};
+  background-color: ${({ theme }) => theme.color.blue100};
+  border-radius: ${({ theme }) => theme.border.radius.large};
+  padding-vertical: ${({ theme }) => theme.spacing.smallGutter}px;
+  padding-horizontal: ${({ theme }) => theme.spacing.smallGutter}px;
+  flex-direction: row;
+  justify-content: space-between;
+`
+
+const ActionBarItem = styled.TouchableHighlight`
+  border-radius: ${({ theme }) => theme.border.radius.standard};
+  padding-vertical: ${({ theme }) => theme.spacing.p2}px;
+  padding-horizontal: ${({ theme }) => theme.spacing[1]}px;
+`
+
+const ItemWrapper = styled.View<{ wrapItem?: boolean }>`
+  flex-direction: ${({ wrapItem }) => (wrapItem ? 'column' : 'row')};
+  flex: 1;
+  justify-content: center;
+  align-items: center;
+  gap: 6px;
+`
+
+const LoaderWrapper = styled.View`
+  align-items: center;
+  justify-content: center;
+  flex: 1;
+`
+
+export const ActionBar = ({
+  onClickStar,
+  onClickArchive,
+  onClickMarkAsRead,
+  loading = false,
+}: {
+  onClickStar: () => void
+  onClickArchive: () => void
+  onClickMarkAsRead: () => void
```

Comment:
If we want to make this a reusable component we can remove these actions from it and extract it, I asked Daði and he did not think we would use it in another places. I wanted to try to add as little code I could to the inbox.tsx file since it is already huge so I took this approach ☺️